packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -444,9 +444,10 @@ Future<gmaps.Icon?> _gmIconFromBitmapDescriptor(
   return icon;
 }
 
-// Computes the options for a new [gmaps.Marker] from an incoming set of options
-// [marker], and the existing marker registered with the map: [currentMarker].
-Future<gmaps.MarkerOptions> _markerOptionsFromMarker(
+/// Computes the options for a new [gmaps.Marker] from an incoming set of options
+/// [marker], and the existing marker registered with the map: [currentMarker].
+@visibleForTesting
```

Comment:
This feels a little weird, and I think maybe all the methods here could be static methods of a class

Reply:
Yeah, this file started up being small, then grew to epic proportions. It should eventually disappear, but there's still some "stringly typed" API coming from the core plugin (everything that is backed by a JSON object, for example).

Thanks for adding a convert_test!

Reply:
>this file started up being small, then grew to epic proportions. It should eventually disappear
Haha, classic!

Reply:
In order to not have to modify this method, I'd suggest you add the test to `markers_test.dart`, which is calling all the conversion methods when doing `controller.addMarkers`.

See this:

* https://github.com/flutter/packages/blob/c77ab99ab8385690f31e72b6a66f01f9c75dd60e/packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/markers_test.dart#L57-L95